app/src/main/res/values/strings.xml

```diff
@@ -64,17 +64,17 @@
     <string name="importOptionApplicationButton">Use external app</string>
 
     <string name="about">About</string>
-    <string name="app_copyright_fmt" translatable="false" tools:ignore="PluralsCandidate">Copyright 2019–<xliff:g>%d</xliff:g> Sylvia van Os.</string>
-    <string name="app_copyright_old">Based on Loyalty Card Keychain, copyright 2016–2020 Branden Archer.</string>
-    <string name="app_license">Copylefted libre software, licensed GPLv3+.</string>
+    <string name="app_copyright_fmt" tools:ignore="PluralsCandidate">Copyright © 2019–<xliff:g>%d</xliff:g> Sylvia van Os.</string>
+    <string name="app_copyright_old">Based on Loyalty Card Keychain\ncopyright © 2016–2020 Branden Archer.</string>
+    <string name="app_license">🄯 Copylefted libre software, licensed GPLv3+.</string>
     <string name="about_title_fmt">About <xliff:g id="app_name">%s</xliff:g></string>
-    <string name="debug_version_fmt">Version: <xliff:g id="version">%s</xliff:g></string>
+    <string name="debug_version_fmt"><big>Version: <xliff:g id="version">%s</xliff:g></big></string>
     <string name="app_revision_fmt">Revision Info: <xliff:g id="app_revision_url">%s</xliff:g></string>
-    <string name="app_libraries"><xliff:g id="app_name">%s</xliff:g> uses the following third-party libraries: <xliff:g id="app_libraries_list">%s</xliff:g></string>
-    <string name="app_resources"><xliff:g id="app_name">%s</xliff:g> uses the following third-party resources: <xliff:g id="app_resources_list">%s</xliff:g></string>
+    <string name="app_libraries">Libre software libraries: <xliff:g id="app_libraries_list">%s</xliff:g></string>
+    <string name="app_resources">Libre software resources: <xliff:g id="app_resources_list">%s</xliff:g></string>
```

Comment:
Resources are things like images, not software.

Reply:
But freely licensed?

Reply:
Yeah, I don't use anything non-free. Right now the only resources are Google's official Android icons (Apache2) anyway

Reply:
Would be nice with all the licenses added, but this is a selling point.

Reply:
I can change the code to show the license info for each entry. Maybe we can then drop this translation change?

Reply:
![image](https://user-images.githubusercontent.com/1885159/113605843-7e418680-9647-11eb-932a-2f5a812881fc.png)

Better?

Reply:
Yes. :)